catkin_tools/execution/executor.py

```diff
@@ -348,6 +362,7 @@ def execute_jobs(
 def run_until_complete(coroutine):
     # Get event loop
     loop = get_loop()
+    loop.slow_callback_duration = 1.0
```

Comment:
The default is 100ms. Many callbacks take upwards of 300ms, which causes Trollius to generate warnings.